### PR TITLE
docs: readme points to docs.prowler.com to learn everything

### PR DESCRIPTION
**Pull request by jfagoagas**
### Description

All the documentation is at https://docs.prowler.com.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by toniblyx — CHANGES_REQUESTED** [review]
Install, Usage, Tutorials and Developer Guide is at [https://docs.prowler.com/](https://docs.prowler.com/)

Remove v2 documentation section.

File: README.md [hunk 1]
```diff
@@ -41,7 +41,15 @@
 
 # Description
 
-`Prowler` is an Open Source security tool to perform AWS, GCP and Azure security best practices assessments, audits, incident response, continuous monitoring, hardening and forensics readiness.
+**Prowler** is an Open Source security tool to perform AWS, Azure, Google Cloud and Kubernetes security best practices assessments, audits, incident response, continuous monitoring, hardening and forensics readiness, and also remediations! We have Prowler CLI (Command Line Interface) that we call Prowler Open Source and a service on top of it that we call <a href="https://prowler.com">Prowler SaaS</a>.
+
+## Prowler CLI
```
Comment (jfagoagas) [inline README.md]:
```suggestion
## Prowler CLI
```

Comment (jfagoagas) [reply README.md]: The space was strange, not the one needed by markdown.

File: contrib/cloudshell/cloudshell-installation.sh [hunk 2]
```diff
@@ -4,5 +4,5 @@ sudo bash
 adduser prowler
 su prowler
 pip install prowler
-cd /tmp || exit
+cd /tmp
```
Comment (jfagoagas) [inline contrib/cloudshell/cloudshell-installation.sh:7]: Why are you removing the `|| exit` is better to have that not to break anything in such a case that is not present.

File: docs/index.md [hunk 3]
```diff
@@ -124,12 +130,12 @@ Prowler is available as a project in [PyPI](https://pypi.org/project/prowler/),
     adduser prowler
     su prowler
     pip install prowler
-    cd /tmp || exit
+    cd /tmp
```
Comment (jfagoagas) [inline docs/index.md:139]: The same as above.

File: docs/tutorials/aws/cloudshell.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ sudo bash
 adduser prowler
 su prowler
 pip install prowler
-cd /tmp || exit
+cd /tmp
```
Comment (jfagoagas) [inline docs/tutorials/aws/cloudshell.md:10]: Same as above.

**Review by jfagoagas — COMMENTED** [review]
Thanks for improving this @sergargar!!

Could you please add in the Containers section that the following container images are available too?
- `v3-latest`
- `v3-stable`

Thanks!

File: contrib/cloudshell/cloudshell-installation.sh (hunk 2 above)
Comment (toniblyx) [reply contrib/cloudshell/cloudshell-installation.sh:7]: /tmp is present always in this case, it is not harmful at all to leave it or remove it

**Review by toniblyx — APPROVED** [review]

**Pull request merged**